.travis.yml

```diff
@@ -22,13 +22,19 @@ env:
     - DB=pgsql MOODLE_BRANCH=MOODLE_27_STABLE
     - DB=pgsql MOODLE_BRANCH=MOODLE_28_STABLE
     - DB=pgsql MOODLE_BRANCH=MOODLE_29_STABLE
-    - DB=pgsql MOODLE_BRANCH=MOODLE_30_STABLE
```

Comment:
I think this could be left as it is as 3.0 still supports 5.6 https://docs.moodle.org/dev/Moodle_3.0_release_notes#Server_requirements